pkgs/development/libraries/science/benchmark/papi/5.7.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation {
+  version = "5.7.0";
+  pname = "papi";
+
+  src = fetchurl {
+    url = "https://bitbucket.org/icl/papi/get/papi-5-7-0-t.tar.gz";
+    sha256 = "1i7gin71rm00gxvfw0fgfglm58y0zks04frzw01fh45x6n5p8rk4";
+  };
+
+  preConfigure = ''
+    cd src
+  '';
+
+  doCheck = true;
+  checkTarget = "test";
+
+  meta = with lib; {
+    homepage = "https://icl.utk.edu/papi/";
+    description = "PAPI provides the tool designer and application engineer with a consistent interface and methodology for use of the performance counter hardware found in most major microprocessors";
```

```suggestion
    description = "PAPI provides the tool designer and application engineer use of the performance counter hardware found in most major microprocessors";
```

This needs some shorting for description. It would be fine for longDescription